### PR TITLE
<Description> : Adding CGPoint Tests : For adding two CGPoints and calculating the distance between the two.

### PR DESCRIPTION
**Pull request by Khalian**
<Type> : feature/test

### CODE REVIEW

File: Sources/CGPointExtensions.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+extension CGPoint {
+    
+    /// EZSE: Converts angle degrees to radians.
+    public static func + (this: CGPoint, that: CGPoint) -> CGPoint {
+        return CGPoint(x: this.x + that.x, y: this.y + that.y)
+    }
+    
+    /// EZSE: Calculates the distance between two CG Points. 
+    /// Inspired by : http://stackoverflow.com/questions/1906511/how-to-find-the-distance-between-two-cg-points
+    public static func absDistance(this: CGPoint, that: CGPoint) -> CGFloat {
```
Comment (piv199) [inline Sources/CGPointExtensions.swift]: why call it absDistance if distance is already abs'ed?

Comment (Khalian) [inline Sources/CGPointExtensions.swift]: should i just name it distance?

Comment (piv199) [inline Sources/CGPointExtensions.swift]: @Khalian I'm for it

Comment (piv199) [inline Sources/CGPointExtensions.swift]: also this/that is not appropriative - use from/to or start/end

**Pull request merged**